uktrade/data-hub-frontend
# Add contact loop to the interaction form

## Description of change

Add contact look to the interaction form allowing users for uninterrupted journey.

## Test instructions

1. Go to [the new interaction form](https://data-hub-fro-contact-lo-cyaipi.herokuapp.com/interactions/create?company=5bb239e4-b227-4a84-bc2e-3420fc129447)
2. Chose the interaction type
3. Fill some fields
4. Click `add a new contact` from the link within the `Contact(s)` field
5. Create new contact
6. You should be redirected to the interaction form with all the form values restored
